diff --git a/meson.build b/meson.build
index 6dd75af56e..a1023aa841 100644
--- a/meson.build
+++ b/meson.build
@@ -1856,7 +1856,7 @@ client_api_version = major + '.' + minor + '.0'
 libmpv = library('mpv', sources, dependencies: dependencies, gnu_symbol_visibility: 'hidden',
                  include_directories: incdir_public,
                  link_args: cc.get_supported_link_arguments(['-Wl,-Bsymbolic']),
-                 version: client_api_version, install: get_option('libmpv'),
+                 install: get_option('libmpv'),
                  build_by_default: get_option('libmpv'))
 
 
